Model/Token.php

```diff
@@ -74,12 +83,17 @@ public function getToken()
 
             $result = json_decode(file_get_contents($certCaptureConfig['url'], false, $context), true);
```

Comment:
It seems like there should be some sort of error handling around this. What happens if the Avalara URL times out or returns a non-JSON response?